projects/1.16/assets/materialis/materialis/lang/zh_cn.json

```
@@ -152,42 +219,133 @@
+  "modifier.materialis.pipe_wrenching": "拧管道",
+  "modifier.materialis.pipe_wrenching.description": "工具可以设置更好的管道",
+  "modifier.materialis.pipe_wrenching.flavor": "真漂亮啊！",
+  "modifier.materialis.pixiecaller": "精灵召唤者",
+  "modifier.materialis.pixiecaller.description": "当你受到伤害时有几率生成一只小精灵",
+  "modifier.materialis.pixiecaller.flavor": "无精灵无真相",
   "modifier.materialis.polished": "抛光",
+  "modifier.materialis.polished.attack_damage": "抛光攻击伤害",
   "modifier.materialis.polished.description": "耐久度越高，攻击伤害越高",
   "modifier.materialis.polished.flavor": "我能从里面看到我的脸！",
+  "modifier.materialis.power_hungry": "电老虎",
+  "modifier.materialis.power_hungry.description": "每级工具每损失1点耐久值消耗100RF，如果没有能量就看起来无法正常运作",
+  "modifier.materialis.power_hungry.flavor": "自大狂！",
+  "modifier.materialis.processor": "中央处理器",
+  "modifier.materialis.processor.description": "使你的工具在使用术式时处理传感器的输入",
+  "modifier.materialis.processor.flavor": "它是加工出来的吗？",
   "modifier.materialis.psicho_digger": "Psi挖掘者",
   "modifier.materialis.psicho_digger.description": "拥有的Psi越多，采掘速度越快",
   "modifier.materialis.psicho_digger.flavor": "跑，跑远点",
+  "modifier.materialis.psicho_digger.mining_speed": "满Psi时的挖掘速度",
   "modifier.materialis.psicho_killer": "Psi杀手",
-  "modifier.materialis.psicho_killer.description": "拥有的Psi越多，攻击伤害越高",
+  "modifier.materialis.psicho_killer.attack_damage": "满Psi时的攻击伤害",
+  "modifier.materialis.psicho_killer.description": "拥有的psi越多，攻击伤害越高",
   "modifier.materialis.psicho_killer.flavor": "那是什么？",
+  "modifier.materialis.psionizing_radiation": "Psi离辐射",
+  "modifier.materialis.psionizing_radiation.description": "你的工具对psi离子有很强的传导性，适合施放术式，+1术式插槽",
+  "modifier.materialis.psionizing_radiation.flavor": "感觉有点好笑",
+  "modifier.materialis.psionizing_radiation_attack": "Psi衰变（攻击）",
+  "modifier.materialis.psionizing_radiation_attack.description": "使用工具攻击时会施放术式",
+  "modifier.materialis.psionizing_radiation_attack.flavor": "你能感觉到它在发光",
+  "modifier.materialis.psionizing_radiation_break_block": "Psi衰变（破坏方块）",
+  "modifier.materialis.psionizing_radiation_break_block.description": "使用工具破坏方块时会施放术式",
+  "modifier.materialis.psionizing_radiation_break_block.flavor": "你能感觉到它在发光",
+  "modifier.materialis.psionizing_radiation_damage": "Psi衰变（伤害）",
+  "modifier.materialis.psionizing_radiation_damage.description": "受到伤害时工具会释放术式",
+  "modifier.materialis.psionizing_radiation_damage.flavor": "你能感觉到它在发光",
+  "modifier.materialis.psionizing_radiation_detonate": "Psi衰变（引爆）",
+  "modifier.materialis.psionizing_radiation_detonate.description": "当周围有术式引爆时工具会释放术式",
+  "modifier.materialis.psionizing_radiation_detonate.flavor": "你能感觉到它在发光",
+  "modifier.materialis.psionizing_radiation_fire": "Psi衰变（着火）",
+  "modifier.materialis.psionizing_radiation_fire.description": "受到火焰伤害时工具会释放术式",
+  "modifier.materialis.psionizing_radiation_fire.flavor": "你能感觉到它在发光",
+  "modifier.materialis.psionizing_radiation_health": "Psi衰变（低生命值）",
+  "modifier.materialis.psionizing_radiation_health.description": "当你处于低生命值时工具会释放术式",

```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
```suggestion
  "modifier.materialis.psionizing_radiation_health.description": "当你生命低值时工具会释放术式",
```
原文：**Enables the tool to cast spells when your health is low**